app/models/product.rb

```diff
@@ -0,0 +1,5 @@
+class Product < ActiveRecord::Base
+	has_many :product_variants
+	belongs-to :brand
+	belongs_to :category
```

Comment:
This association should be reversed so that the same product can be in multiple categories